docs/operating-scylla/security/security-checklist.rst

```diff
@@ -31,11 +31,9 @@ Encryption on Transit, Client to Node and Node to Node
 Encryption on Transit protects your communication against a 3rd interception on the network connection.
 Configure ScyllaDB to use TLS/SSL for all the connections. Use TLS/SSL to encrypt communication between ScyllaDB nodes and client applications.
 
-.. only:: enterprise
-
-    Starting with version 2023.1.1, you can run ScyllaDB Enterprise on FIPS-enabled Ubuntu, 
-    which uses FIPS 140-2 certified libraries (such as OpenSSL, GnuTLS, and more) and Linux 
-    kernel in FIPS mode.
+You can run ScyllaDB Enterprise on FIPS-enabled Ubuntu, 
```

Comment:
Is it 'ScyllaDB Enterprise' or just 'ScyllaDB' ?

Reply:
Good question.
We do you  use 'ScyllaDB Enterprise'[1]  or ScyllaDB, so both are fine.
https://www.scylladb.com/2024/12/18/why-were-moving-to-a-source-available-license/